### PR TITLE
E2E tests for kubectl commands (api-versions, cluster-info, describe)

### PR DESCRIPTION
**Pull request by mwielgus**
As requested in #10764.

### CODE REVIEW

**Comment by mwielgus** [comment]
cc @jlowdermilk

File: test/e2e/kubectl.go [hunk 1]
```diff
@@ -202,8 +204,144 @@ var _ = Describe("Kubectl client", func() {
...
+			}
+		})
+	})
+
+	Describe("Kubectl cluster-info", func() {
+		It("should check if Kubernetes master services is included in cluster-info", func() {
+			By("validating cluster-info")
+			output := runKubectl("cluster-info")
+			// Can't check exact strings due to terminal controll commands (colors)
+			requiredItems := []string{"Kubernetes master", "KubeDNS", "Heapster", "is running at"}
```
Comment (jlowdermilk) [inline test/e2e/kubectl.go]: I think the only one we can require is `Kubernetes master`. KubeDNS and Heapster are optional, no? If we can assume they are required at least for gce/gke, then this test should include a `SkipUnlessProviderIs("gce", "gke")`

Comment (mwielgus) [inline test/e2e/kubectl.go]: Added `if providerIs("gce", "gke") {...`

File: test/e2e/kubectl.go [hunk 2]
```diff
@@ -202,8 +204,147 @@ var _ = Describe("Kubectl client", func() {
...
+				{"Status:", "Active"}}
+			checkOutput(output, requiredStrings)
+
+			// Quota and limitrange are skipped for now.
+		})
+	})
+
 })
 
+// Checks whether the output splitted by line contains the required elements.
```
Comment (jlowdermilk) [inline test/e2e/kubectl.go]: nit: s/splitted/split

**Comment by jlowdermilk** [comment]
LGTM

File: test/e2e/kubectl.go (hunk 2 above)
Comment (mwielgus) [inline test/e2e/kubectl.go]: fixed

**Comment by jlowdermilk** [comment]
Thanks for adding @mwielgus. Since the other e2e test was marked post-1.0 I suspect we'll need to wait until next week to merge.

**Pull request merged**